psydac/feec/pull_push.py

```diff
@@ -0,0 +1,215 @@
+from sympy import lambdify
```

Comment:
Every function in this file only works for 3D mappings. Maybe we should call it `push_pull_3d.py`?

We definitely need 1D and 2D versions as well.

Further, there are no docstrings...

Reply:
yes we only have the 3d

Reply:
we should keep the same name, and add the other cases later